editor/plugins/tiles/tiles_editor_plugin.cpp

```diff
@@ -158,6 +158,9 @@ void TilesEditorPlugin::_update_editors() {
 	// Update the viewport.
 	CanvasItemEditor::get_singleton()->update_viewport();
 
+	// Make sure the tile set editor is visible if we have one assigned.
+	tileset_editor_button->set_visible(is_visible && tile_set.is_valid());
```

Comment:
I am not sure I understand why this is needed. Wasn't this already working?
Reading what is in make_visible() it seems like this is already done.

Reply:
@groud Sorry, I completely forgot about this, but I remember it now because I'm doing another pass over all the editor plugins at the moment. With regards to this visibility check, it needs both checks to be successful otherwise the tileset button will still be visible even when you aren't highlighting a tileset node.